src/components/tutorials-landing-view/certification-content-card-link/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { type CertificationContentCardLinkProps } from '../types'
+import ContentCardLink from '../content-card-link'
+import consulGraphic from './img/consul.svg'
+import terraformGraphic from './img/terraform.svg'
+import vaultGraphic from './img/vault.svg'
+
+const PRODUCT_SLUGS_TO_BACKGROUND_IMAGES = {
+	consul: {
+		url: consulGraphic,
+		lightOrDark: 'dark',
+	},
+	terraform: {
+		url: terraformGraphic,
+		lightOrDark: 'dark',
+	},
+	vault: {
+		url: vaultGraphic,
+		lightOrDark: 'light',
+	},
+}
+
+const CertificationContentCardLink = ({
+	productSlug,
+	description,
+	href,
+	title,
+}: CertificationContentCardLinkProps) => {
+	const { url, lightOrDark } = PRODUCT_SLUGS_TO_BACKGROUND_IMAGES[productSlug]
```

Comment:
💭 We'll run into a runtime error if we try to use a product not in the `PRODUCT_SLUGS_TO_BACKGROUND_IMAGES` array, but we won't run into a type check error.  No need to address, just a thought I wanted to voice.

Reply:
I'm probably going to rework the types on here once we plug in the data, so I'll make sure this is top of mind in that work.